meta-lmp-base/recipes-containers/skopeo/skopeo/0001-cmd-copy-max-parallel-pulls-param.patch

```diff
@@ -0,0 +1,41 @@
+From 897767edb5dda313483ba5496138230eb5383e3a Mon Sep 17 00:00:00 2001
+From: Mike Sul <mike.sul@foundries.io>
+Date: Tue, 6 Jun 2023 20:33:48 +0200
+Subject: [PATCH] [fio up]: cmd: copy: Add param to specify max parallel pulls
+
```

Comment:
Please add a line with 'Upstream-Status: Pending', so we know this can eventually be sent upstream.

Reply:
I added and updated the branch, since it is hosted here.

Reply:
> I added and updated the branch, since it is hosted here.

Thanks.